src/modules/tests/tests.effects.ts

```diff
@@ -205,6 +205,14 @@ export class TestsEffects {
         startTestAction.category === TestCategory.DE) {
         arrayOfActions.push(new InitializeVehicleChecksCatD(startTestAction.category));
```

Comment:
Need to make sure we are setting business details correctly and dispatching the right actions for that.